bioimageio/core/predict.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import sys
+
+import numpy as np
+import torch.cuda
+import xarray as xr
+
+from bioimageio.core.prediction_pipeline import create_prediction_pipeline
+from bioimageio.spec import import_package
```

Comment:
I guess we can replace this with `load_node` once the functionality is unified and then this can work both with zip and yaml, which is nice!

Reply:
`load_resource_description` now